### ISSUE TITLE
Shared listening - Radio with friends

### ISSUE THREAD

**Issue by peperunas** [navidrome/navidrome#1551]
First of all, great work!

### Is your feature request related to a problem? Please describe.

It would be great to be able to share a link to a simple "web radio". The radio owner can change the songs' queue on the fly and the guests are presented with a splash screen with album art, title and metadata.

**Comment by deluan** [comment]
Hey, sorry for the late reply. This may be possible when we implement the Jukebox mode (#364), and instead of having the server output to an audio device attached to the server, it could potentially send the stream to a shoutcast/icecast server or something similar.

We plan to do something similar for Chromecast support, so this is not too far from the current scope, but I don't have plans to work on it anytime soon, as there are currently other priorities in the project.

**Comment by DOFfactory** [comment]
Just found this thread and I would also suggest +1 similar functionality: share playlists with users on the same server. We have 2 accounts on the same navidrome server, but both of us have different 'favourites' lists. Would be great to share it with other users on the same server...

**Comment by peperunas** [comment]
If @deluan agrees, we could change this issue to track the "shared listening" topic and move to #1708 for the specific "shared playing" resolution.

**Comment by max-tet** [comment]
I'd also love this feature and I have a request regarding the specific implementation. I'll just copy what I wrote [here](https://github.com/navidrome/navidrome/issues/748#issuecomment-1264576075).

> Can I just request that the sharing URL which is used for anonymous playback has a different prefix from the rest of the app? Right now, all of the Navidrome UI is at /app/. So maybe anonymous listening could be at /share/ or something like that? I hope this introduces no additional effort.
> 
> The reason is that this would make it very straightforward to setup access control on Portal as described [here](https://docs.getportal.org/routing_and_ac/#access-control).

**Comment by AvMavs** [comment]
Yes, I'd love if this could work too, it'd be amazing to shift all my friends away from other proprietary services as well!

**Comment by laserheat** [comment]
I would like to see this implemented.  People can tune in and listen to the same song at the same time like a radio station.  Also letting people pick music and add it to the queue is a great feature. If no one adds a song, it automatically picks the next song until someone adds to the queue. Something like Streamsicle with a modern interface is what I am after.

**Comment by laserheat** [comment]
Keeping this request alive.  Hopefully, others would like the feature added as well.

**Comment by TacoCake** [comment]
I would also love this feature !

**Comment by matkam7** [comment]
I too would like this feature, especially now that Jukebox is working

**Comment by laserheat** [comment]
Hopefully, we can get it back on the radar :)

**Comment by alfureu** [comment]
bad bot

**Comment by laserheat** [comment]
Agreed, was hoping this would make the features.

**Comment by TesterHere06** [comment]
+
Would be a really amazing feature

**Comment by hatkidchan** [comment]
\+ to it too

I think it's possible to do it in a hacky way by using [Snapcast integration](https://www.navidrome.org/docs/usage/jukebox/#the-mpvcmdtemplate--snapcast-integration) and running, say, `ffmpeg` instead that would just stream to icecast

I could try doing it in the near future, maybe

**Comment by laserheat** [comment]
We need a simple way for all to tap into the same stream; most browsers can
play MP3 streams with no software needed. However, if it were simple, we
all could code it. :)

**Comment by encode42** [comment]
Not stale, listening to music with friends is fun!

**Comment by Viper-0501** [comment]
+1

**Comment by WinterFe** [comment]
+1 - Really want this feature to be added.

**Comment by nozhana** [comment]
+1 same here!! Been looking for a solution for so long. Jellyfin already has this feature implemented.

**Comment by edurenye** [comment]
Would be great also to be able to turn a playlist into a radio.
